instances.py

```diff
@@ -22,7 +22,27 @@ def replying_jokes(posts, reddit):
 					sub1 = submission1.title
 					sub2 = submission1.selftext
 					i.reply("Maverick is here with a joke: "+ sub1 + " " + sub2)
-			alreadyreplied.append(i.id)
+				alreadyreplied.append(i.id)
 	with open(config.FILENAME,"w") as fo:
-		for posts in alreadyreplied:
-			fo.write(posts + "\n")
+		for post in alreadyreplied:
+			fo.write(post + "\n")
+
+def news(posts, reddit):
+	if not os.path.isfile(config.FILENAME):
```

Comment:
For next time, if we reuse this code block, we should declare functions so that code doesn't get duplicated.